arwen/ethapi/ethereumei.go

```diff
@@ -487,15 +487,15 @@ func ethgetExternalCodeSize(context unsafe.Pointer, addressOffset int32) int32 {
 	metering := arwen.GetMeteringContext(context)
 
 	dest, err := runtime.MemLoad(addressOffset, arwen.AddressLen)
-	if withFault(err, context) {
+	if arwen.WithFault(err, context) {
 		return 0
 	}
 
 	gasToUse := metering.GasSchedule().EthAPICost.GetExternalCodeSize
 	metering.UseGas(gasToUse)
 
 	codeSize, err := blockchain.GetCodeSize(dest)
-	if withFault(err, context) {
+	if arwen.WithFault(err, context) {
```

Comment:
as we have to keep the ethereum API, in places where we return 0 / 1 / -1 depending on failure or not, there is no need to set breakPoint and get out of arwen.

Reply:
Changed `WithFault` to be more flexible. Within the Ethereum API, no breakpoints will be set.